src/HasParent.php

```diff
@@ -103,12 +103,9 @@ public function getClassNameForRelationships()
      */
     public function getMorphClass()
     {
-        if ($this->parentHasHasChildrenTrait()) {
```

Comment:
Is there a reason for removing this check?

Reply:
My reasonings are explained here: https://github.com/calebporzio/parental/pull/76